Bug fixes. (#1662)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1019-1-2-corrected-placement-of-marker-when-one-or-both-the-coordinates-is-set-to-zero.md

---
date: 2023-12-14

> title: Corrected placement of marker when one or both the co-ordinates is set to zero

Reviewer: ```suggestion
title: Corrected placement of marker if coordinate value is set to zero
```

product_area: Application enablement & solutions
change_type:
  - value: change-2c7RdTdXo4
    label: Fix
component:
  - value: component-Tl88RYb4A
    label: Digital Twin Manager
build_artifact:
  - value: tc-wYIY0MBDO
    label: dtm
ticket: CTM-995
version: 1019.1.2
---

> The marker is now shown at the center of the map when both or any one of the value of latitude and longitude is "0" for the Location property.

Reviewer: ```suggestion
The marker is now shown at the center of the map when one or both of the values for latitude and longitude is "0" for the Location property.
```

Author: The marker is now shown at the center of the map when one or both of the values for latitude and longitude is "0" for the Location property.Previously, the marker was not centrally displayed, requiring users to manually zoom in to locate it.

> The marker is now shown at the center of the map when both or any one of the value of latitude and longitude is "0" for the Location property.

Reviewer: Where is that? Please provide a context.

Author: The marker is now shown at the center of the map when one or both of the values for latitude and longitude is "0" for the Location property.Previously, the marker was not centrally displayed, requiring users to manually zoom in to locate it.
